#!/usr/bin/env python
PACKAGE = "animat"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("shortage", double_t, 0, "Under this threshold for a physiology variable, there is a shortage", .01, 0, 1)
gen.add("decay_H", double_t, 0, "Hydration decay (in unit/s)", .01, 0, .1)
gen.add("decay_G", double_t, 0, "Glycemia decay (in unit/s)", .01, 0, .1)



exit(gen.generate(PACKAGE, "animat", "PhysiologyParam"))
